docs/user-documentation/content_types.md

```diff
@@ -0,0 +1,126 @@
+1. Now we configure how the field is stored in the Drupal database. For this field type we can select how many values will be allowed. Set this value to "1":
+1. Now we configure how the field is described (including its display label and the help text for when it appears on a form) and constraints on its use. In this case, the field will be required for this Content Type, and will be set to “on” by default: 
+
+![a screenshot of the field settings page](/docs/assets/islandora8_fieldsettings.png)
+
+*Save*. Now we have added our new field: 
+
+![a screenshot of a "Needs Review?" field in the Drupal field UI](/docs/assets/islandora8_newfield.png)
+
+And it appears in the ingest form when we try to create a new repository object:
+
+![a screenshot of a "Needs Review?" field appearing at the bottom of a new node form](/docs/assets/islandora8_newfieldinform.png)
+
+!!! tip New fields, with the exception of Typed Relation fields, are not automatically indexed in Fedora and the triple-store. Update the Content Type's RDF Mapping [link that section of the docs] to enable indexing the field. 
+
+!!! tip To add new behavior based on the results of this new field, check out [link to Context docs]
+
+### Change the Form Display
+
+Now let’s move our new field to a different part of the form. In the Admin menu, return to Structure > Content Types and find the Repository Item content type again. Select Manage form display.
+
+1. All of the fields in this content type are available, in a list, with a simple drag-and-place UI. Drag the *Needs Review?* Field to the top of the form. We can also change the way the Boolean options are displayed, with radio buttons as opposed to a single checkbox. Different display options will be available depending on field type. For more information, please check out [List of Drupal 8 FieldTypes, FieldWidgets, and FieldFormatters](https://www.drupal.org/docs/8/api/entity-api/fieldtypes-fieldwidgets-and-fieldformatters)
+1. Save. When creating a new Repository Item, the *Needs Review?* Field appears at the top, as a set of radio buttons.
+
+### Change the Content Display
+
+Finally, let’s change how the results of this field are displayed. As things stand now, our new field shows up at the bottom of repository object pages:
+
+![a screenshot of a "Needs Review?" field in the node display](/docs/assets/islandora8_fieldindisplay.png)
+
+In the Admin menu, return to Structure > Content Types and find the Repository Item content type again. Select Manage display.
+
+1. Find the *Needs Review?* We can change how the field title is display (inline/above/hidden entirely) and replace the options displayed with variations on a binary choice (yes/no, enabled/disabled, checkmark/X) or hide the field completely. 
+1. We can also drag the field into the Disabled section so that neither its label or its contents appear in the display, although the area saved on the node.
```

Comment:
Change: `although the area saved on the node`
To: although the **field is** saved on the node